
int32 x		# x of position
int32 y		# y of position
int32 w		# width
int32 h		# height
